tidb-cloud/tidb-cloud-security-password-authentication.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Password Authentication
+summary: Learn how to sign up, sign in and sign out.
+---
+
+# Password Authentication
+
+Users may login in TiDB Cloud Console via Email and password. It mainly includes steps:
+
+- Step1: sign up
+- Step2: sign in and sign out
+- Step3: Password policy
+- Step4: Reset and change Password
+- Step5: Enables MFA (optional)
+
+## Setp1: sign up
+
+To sign up for a TiDB Cloud account, please click on the [TiDB Cloud homepage](https://tidbcloud.com/signup).
+
+1. Enter the following information:
+
+2. Read Privacy Policy and Services Agreement
+
+3. Check the box to agree **I agree to the Privacy Policy and Services Agreement**
+
+4. Click **Sign up** to complete your account registration
+
+You will receive a verification email for TiDB Cloud, when you check your Email address and confirm in order to complete the whole registration process.
```

Comment:
```suggestion
```

Reply:
Because this document is about Password Authentication, step-by-step instructions about sign-up, sign-in, and sign-out are not needed for it.